lib/cfd_trading/cfd_order_detail.dart

```diff
@@ -170,6 +181,7 @@ class _CfdOrderDetailState extends State<CfdOrderDetail> {
                             Visibility(
                               visible: confirm,
                               child: SubmitButton(
+                                  isButtonDisabled: disableActionButton,
```

Comment:
nit: no need for this variable you could simply add `!channel.isAvailable()` here.